src/App.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
-import Calculator from './components/Calculator';
+import Home from './pages/Home';
+import Navbar from './components/Navbar';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.